packages/web-comment/src/main.njk

```diff
@@ -0,0 +1,67 @@
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% block head %}
+  {% include "includes/head.njk" %}
+{% endblock %}
+
+{% block header %}
+  {% include "includes/cookie-banner.njk" %}
+
+  {{ govukHeader({
+    homepageUrl: "https://www.gov.uk/",
+    serviceName: 'Comment on a planning decision',
+    serviceUrl: "/",
+    containerClasses: "govuk-width-container"
+  }) }}
+{% endblock %}
+
+{% block bodyStart %}
+  {% include "includes/body-start.njk" %}
+{% endblock %}
+
+{% set mainClasses = mainClasses | default("govuk-main-wrapper--auto-spacing") %}
+
+{% block main %}
+  <div class="govuk-width-container govuk-body {{ containerClasses }}">
+
+    <main id="main-content" role="main"{% if mainLang %} lang="{{ mainLang }}"{% endif %}>
+      {% block phaseBanner %}
+        {{ govukPhaseBanner({
+          tag: {
+            text: "beta"
+          },
+          html: 'This is a beta service – your <a class="govuk-link" data-cy="Feedback" href="https://forms.office.com/Pages/ResponsePage.aspx?id=mN94WIhvq0iTIpmM5VcIjVqzqAxXAi1LghAWTH6Y3OJUOFg4UFdEUThGTlU3S0hFUTlERVYwMVRLTy4u">feedback</a> will help us to improve it.'
+        }) }}
+      {% endblock %}
+
+      {% block backButton %}
+        {{ govukBackLink({
+          text: "Back",
+          href: navigation[1],
```

Comment:
don't think navigation is defined so this just results in an href of '#'

Reply:
Just resolving for now to get the initial merge done then we can pick these points back up